cli/src/main/java/io/quarkiverse/tekton/cli/pipeline/AbstractPipelineCommand.java

```diff
@@ -28,15 +28,28 @@ public abstract class AbstractPipelineCommand extends GenerationBaseCommand {
     @Inject
     KubernetesClient kubernetesClient;
 
+    void checkNamespace() {
+        // If the argument of the CLI command --namespace has been provided, we use it, otherwise we check
+        // if the namespace has been set using the kubernetes context and config
+        if (namespace.isPresent()) {
+            Clients.setNamespace(namespace.get());
+        } else {
+            Clients.setNamespace(namespace
+                    .or(() -> Optional.ofNullable(Clients.kubernetes().getNamespace()))
+                    .orElseThrow(() -> new IllegalArgumentException(
+                            "No user's namespace provided using the kubecontext or CLI command's argument: --namespace")));
+        }
```

Comment:
```suggestion
        Clients.setNamespace(namespace
                .or(() -> Optional.ofNullable(Clients.kubernetes().getNamespace()))
                .orElseThrow(() -> new IllegalArgumentException(
                        "No user's namespace provided using the kubecontext or CLI command's argument: --namespace")));
```

Reply:
The if/else here is redundant as the else part also checks if namespace is present.